web/vite.config.ts

```diff
@@ -11,12 +11,17 @@ export default defineConfig({
 	},
 	server: {
 		proxy: {
-			"/api": {
-				target: "http://localhost:1337",
+			"/api/v1": {
+				target: "http://localhost:3000",
```

Comment:
selfish request.  Can we use a different port?  One of my work projects uses 3000 by default :P

Reply:
Done! Updated Fe to `5000` and BE to `1337`.